app/models/game.rb

```diff
@@ -2,4 +2,9 @@ class Game < ActiveRecord::Base
   belongs_to :white_user, :class_name => 'User'
   belongs_to :black_user, :class_name => 'User'
   has_many :pieces
+
+  scope :needing_second_player, -> { where(black_user_id: nil) }
```

Comment:
@jacobwgillespie  is this another instance where we don't need to include "_id" when calling the user?